platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/DefaultProjectManager.java

```diff
@@ -1019,8 +1050,8 @@ private void deleteProjectFromWorkspace(String wsId, String projectName) throws
 
         try {
             HttpJsonHelper.request(null, link);
-        } catch (IOException | ApiException e) {
-            throw new ServerException(e);
+        } catch (IOException | ApiException exception) {
```

Comment:
Just interesting: what is wrong with `e` ? 
In this case i prefer to use either `ex` or separated `ioEx` and `apiEx` names, but i don't think that `expection` is better than `e`